staff/sys/migrate-vm

```diff
@@ -76,25 +75,25 @@ def copy_disk_image(vm, num_bytes):
     proc.wait()
 
 
-def verify_disks(vm):
-    proc = Popen(['openssl', 'sha1', '/dev/vg/' + vm], stdout=PIPE)
-    _, stdout, _ = client.exec_command('openssl sha1 /dev/vg/' + vm)
+def verify_disks(oldpath, newpath):
+    proc = Popen(['openssl', 'sha1', newpath], stdout=PIPE)
+    _, stdout, _ = exec_command('openssl sha1 ' + oldpath)
 
     original = stdout.read().decode('ascii').strip()
     print(bold('Original SHA1 (on remote server): ') + original)
 
     copied = proc.stdout.read().decode('ascii').strip()
     print(bold('New SHA1 (on this server): ') + copied)
 
-    if original != copied:
+    # Pull out the hash string since file names may not match
+    if original.split()[1] != copied.split()[1]:
```

Comment:
I did worry about that and originally regex'd the two strings, but then I thought "Well, it's printed right above, won't someone notice if the output isn't a SHA1 sum?"

But I think you're right. If it says "success", there's the chance someone will miss some subtle problem with the output, or will just skip reading the hashes altogether.